src/graphql/admin/root/mutation/account-update-status.ts

```diff
@@ -17,7 +18,7 @@ const UserUpdateStatusInput = new GT.Input({
 })
 
 const UserUpdateStatusMutation = GT.Field({
-  type: GT.NonNull(UserDetailPayload),
+  type: GT.NonNull(AccountDetailPayload),
```

Comment:
please update UserUpdateStatusMutation to AccountUpdateStatusMutation and the same for UserUpdateStatusInput